Cabal/tests/ParserTests/regressions/issue-5055.format

```diff
@@ -0,0 +1,25 @@
+cabal-version: >=2.0
+name: issue
+version: 5055
+license: BSD3
+synopsis: no type in all branches
+description:
+    no type in all branches.
+category: Test
+build-type: Simple
+
+executable flag-test-exe
+    main-is: FirstMain.hs
+    scope: unknown
+    default-language: Haskell2010
+    build-depends:
+        base >=4.8 && <5
+
+test-suite flag-cabal-test
+    type: exitcode-stdio-1.0
+    main-is: SecondMain.hs
+    default-language: Haskell2010
+    build-depends:
+        base >=4.8 && <5
+    
+    if os(windows)
```

Comment:
A pretty printer bug?

Reply:
... as this is "bug-compatible" behaviour, with `cabal-install-2.0`. `cabal-2.0` formats the input as

```
name: issue
version: 5055
cabal-version: >=2.0
build-type: Simple
license: BSD3
synopsis: no type in all branches
description:
    no type in all branches.
category: Test

executable  flag-test-exe
    main-is: FirstMain.hs

test-suite  flag-cabal-test
    type: exitcode-stdio-1.0
    main-is: SecondMain.hs
    build-depends:
        base >=4.8 && <5
    default-language: Haskell2010
```

There is no conditional section at all

Reply:
OK, fine, let's merge as is.